### ISSUE TITLE
Distinctive Identifier association mediated by CDM server

### ISSUE THREAD

**Issue by lxgr** [w3c/encrypted-media#390]
It seems that while the spec does not allow a CDM to share origin names (or associable equivalents) with its central server, and also prohibits Distinctive Identifiers that could be associated by colluding applications, the possibility of applications using the CDMs central server to associate Distinctive Identifiers is not explicitly mentioned.

Assuming that this is intentional, would it make sense to explicitly mention this in the section on privacy concerns, i.e. that the specification does not protect against associations by different applications, mediated by the CDM central server?

**Comment by ddorwin** [comment]
Yes, that first paragraph is roughly accurate, though things like a "central server" are not specified.

Distinctive Permanent Identifiers must not be exposed to the application or origin, and thus could only be exposed to application- and origin-independent server(s) via some other channel, which could be to a "central server." In the [Direct Individualization](https://w3c.github.io/encrypted-media/#direct-individualization) case, the server would likely respond with a Distinctive Identifier. The user agent must not expose origin- or application-related information (i.e., to the server) during this process, so while the server can associate the Distinctive Permanent Identifier with the Distinctive Identifiers, it should not know for which origins they were issued.

As you note, there is nothing in the spec that prevents applications and the central server from colluding, though this would likely violate the intent and spirit of the spec. We could add text prohibiting this, but it may be outside the scope of such a spec since it is not something that can be enforced by the client. (The spec does, however, have similar text in the last paragraph of the [App-Assisted Individualization](https://w3c.github.io/encrypted-media/#app-assisted-individualization) section.) Short of that, acknowledging this - if it's not already - seems reasonable.

**Comment by mwatson2** [comment]
I agree with the comments above, but think there is a more general issue in that the Privacy section could better explain how the (non-normative) concerns listed there are mitigated by the requirements of the specification. This is a bigger job than adding a simply note about this one case, so I suggest we mark this one for VNext.

**Comment by jdsmith3000** [comment]
I agree with marking this VNext.
